docs/README-for-doc.zonemaster.net.md

```diff
@@ -50,4 +50,4 @@ The public documentation is divided into the following main categories:
 [Zonemaster.net]:                                   https://zonemaster.net
 
 <!-- Location on doc.zonemaster.net: --> 
-[Zonemaster-logo]:                                  ../zonemaster_logo_2021_color.png
+[Zonemaster-logo]:                                  ../zonemaster_logo_2021.png
```

Comment:
What is the purpose of this change? I think it is better to keep the same name as we have in https://github.com/zonemaster/zonemaster/tree/master/assets/images